src/main/java/duke/Event.java

```diff
@@ -0,0 +1,82 @@
+package duke;
+
+import java.time.LocalDate;
+
+/**
+ * Constructs an event task.
+ */
+public class Event extends Task {
+
+    protected LocalDate from;
+    protected LocalDate to;
+
+    /**
+     * Constructs an Event with a description, start time, and end time.
+     *
+     * @param description The description of the event.
+     * @param from        The start time of the event.
+     * @param to          The end time of the event.
+     * @throws DukeException If the date strings are not in a valid format.
+     */
+    public Event(String description, String from, String to) throws DukeException {
+        super(description);
+        this.from = DateChecker.parseDate(from);
+        this.to = DateChecker.parseDate(to);
+    }
+
+    /**
+     * Returns the start time of the event.
+     *
+     * @return The start time of the event.
+     */
+    public LocalDate getStart() {
```

Comment:
Method names could be slightly more detailed to reflect the return value. getStartDate() and getEndDate() for example.